designs/developer/operator-virtualization-45/vm-sidepanel-resources.md

```diff
@@ -0,0 +1,8 @@
+---
+parent: Developer
+version: 4.5
+---
+
+## Design for the Resource tab for VM-based resources on topology
+
+![Design for vm-based resources resource tab](designs/developer/operator-virtualization-45/img/vm-topology-sidepanel-resource.png)
```

Comment:
@Veethika  The path to this images seems incorrect. I think you just want to have "img/vm-topology-sidepanel-resource.png"

Reply:
Everything else looks great!